Sources/Dependencies/DependencyKey.swift

```diff
@@ -118,12 +118,9 @@ public protocol DependencyKey: TestDependencyKey {
 /// return a default value suitable for Xcode previews, or the ``testValue``, if left unimplemented.
 ///
 /// See ``DependencyKey`` to define a static, default value for the live application.
-public protocol TestDependencyKey {
+public protocol TestDependencyKey: Sendable where Value: Sendable {
```

Comment:
We don't actually need the key type to conform, just the value, so we can move that constraint to the associated type.
```suggestion
public protocol TestDependencyKey {
```